site/docs/v1/tech/events-webhooks/events.adoc

```diff
@@ -27,13 +27,21 @@ These are the events that FusionAuth generates that can be optionally consumed b
 * <<JWT Refresh>> - when an access token is refreshed using a refresh token
 * <<JWT Refresh Token Revoke>> - when a refresh token (or multiple tokens) are revoked
 
+=== Tenant Scoped Events
+
+Tenant scoped events are generated for all applications in a tenant or for none of them.
+
+All user events are tenant scoped because a user is a tenant scoped entity. For example, the `user.delete`, `user.create`, `user.update`, and `user.deactivate` events are all tenant scoped.
+
+A tenant scoped event can, however contain an `applicationId` which can be used to filter events when received. One example is `user.registration.create`. 
+
 === Application Scoped Events
 
 A few events can be generated for one or more specified applications, or for all applications within a tenant. 
 
 [WARNING]
 ====
-The ability to limit the generation of an event for only certain applications is legacy functionality and may be modified in the future. You almost certainly want to use a tenant scoped event and optionally filter on the `applicationId` when consuming the event.
+The ability to limit the generation of an event for only certain applications is legacy functionality and may be modified in the future. You almost certainly want to enable events at the tenant level and optionally filter on the `applicationId` when consuming the event.
```

Comment:
> You almost certainly want to enable events at the tenant level and optionally

Not sure what that means. You always need to enable an event at the tenant level if you want to receive it.

Reply:
Okay, tried to clarify this.